storm-client/src/jvm/org/apache/storm/cluster/StormClusterStateImpl.java

```diff
@@ -423,32 +424,35 @@ public void supervisorHeartbeat(String supervisorId, SupervisorInfo info) {
     }
 
     /**
-     * if znode exists and to be not on?, delete; if exists and on?, do nothing; if not exists and to be on?, create; if not exists and not on?, do nothing;
+     * if znode exists and timestamp is 0?, delete; if exists and timestamp is larger than 0?, do nothing;
```

Comment:
Nit: What are the question marks after 0 for?

Reply:
The question mark on the `on` in the old comment appears to be left-over from the clojure translation where the on argument was spelled `on?`.

It should go away in any new comment.